strategies/test_only/enzyme_end_to_end_multipair_generic_routing.py

```diff
@@ -0,0 +1,142 @@
+"""Dummy strategy used in Enzyme end-to-end tests.
+
+This is almost an exact replica of .enzyme_end_to_end_multipair, 
+except it uses a different trade_routing value.
+
+Here, trade_routing is a list, not a single value
+"""
+import datetime
+from typing import Dict, List
+
+import pandas as pd
+
+from tradeexecutor.strategy.cycle import CycleDuration
+from tradeexecutor.strategy.default_routing_options import TradeRouting
+from tradeexecutor.strategy.execution_context import ExecutionContext
+from tradeexecutor.strategy.pricing_model import PricingModel
+from tradeexecutor.strategy.trading_strategy_universe import load_all_data, TradingStrategyUniverse
+from tradeexecutor.strategy.universe_model import UniverseOptions
+from tradingstrategy.client import Client, BaseClient
+from tradingstrategy.pair import DEXPair, HumanReadableTradingPairDescription
+from tradingstrategy.testing.uniswap_v2_mock_client import UniswapV2MockClient
+from tradingstrategy.timebucket import TimeBucket
+from tradingstrategy.universe import Universe
+from tradingstrategy.chain import ChainId
+
+from tradeexecutor.state.state import State
+from tradeexecutor.state.trade import TradeExecution
+from tradeexecutor.strategy.strategy_type import StrategyType
+from tradeexecutor.strategy.pandas_trader.position_manager import PositionManager
+from tradeexecutor.strategy.reserve_currency import ReserveCurrency
+from tradeexecutor.strategy.trading_strategy_universe import translate_trading_pair
+from tradeexecutor.testing.generic_mock_client import GenericMockClient
+
+
+trading_strategy_engine_version = "0.3"
+trading_strategy_type = StrategyType.managed_positions
+trading_strategy_cycle = CycleDuration.cycle_1s
+reserve_currency = ReserveCurrency.usdc
+
+trade_routing = [TradeRouting.user_supplied_routing_model_uniswap_v2, TradeRouting.user_supplied_routing_model_uniswap_v3]
```

Comment:
Make this `TradeRouting.generic_routing` instead. Then it just loads all supported routing models and the user does not need to care the underlying routing at all.